testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCoercionOnUnpartitionedTable.java

```diff
@@ -183,6 +198,13 @@ public void testHiveCoercionWithDifferentTimestampPrecision()
         doTestHiveCoercionWithDifferentTimestampPrecision(HIVE_TIMESTAMP_COERCION_ORC);
     }
 
+    @Requires(ParquetRequirements.class)
```

Comment:
Can we enable the `testHiveCoercionWithDifferentTimestampPrecision()` for parquet format ?